backend/build.gradle.kts

```
@@ -23,21 +23,26 @@ dependencies {
     annotationProcessor("io.micronaut.security:micronaut-security-annotations")
     annotationProcessor("io.micronaut.data:micronaut-data-processor")
     annotationProcessor("io.micronaut:micronaut-inject-java")
+    annotationProcessor("io.micronaut.validation:micronaut-validation-processor")
     // When.MAYBE warning fix
     annotationProcessor("com.google.code.findbugs:jsr305:3.0.2")
     implementation("ch.qos.logback:logback-classic:1.4.9")
+    implementation("org.slf4j:slf4j-api:2.0.7")
     implementation("com.google.code.findbugs:jsr305:3.0.2")
     implementation("io.micronaut.security:micronaut-security-jwt")
     implementation("io.micronaut.reactor:micronaut-reactor")
     implementation("io.micronaut:micronaut-runtime")
     implementation("io.micronaut.data:micronaut-data-jdbc")
-    implementation("io.micronaut.flyway:micronaut-flyway")
+    // https://github.com/flyway/flyway/issues/3651
+    // 6.0.0-M2 uses 9.16.1 which is before bug introduced in flyway
+    implementation("io.micronaut.flyway:micronaut-flyway:6.0.0-M2")
```

Even while we are overriding the version here, the logging is still broken (as visible in the latest actions run https://github.com/devcordde/devmarkt-backend/actions/runs/5790068436/job/15692508119?pr=216 Section `Stop backend`)

---

well, as far as I can trust my eyes, there are logs and they should be very complete

---

Logs are present, just not formatted in the colorized way of the others.

---

But the problem is, that they are out of order. We have 2 flyway blocks and 2 other blocks. If you look at the timestamps of the different blocks, you notice that they are not ordered.

---

that's interesting

---

After some investigation, I noticed that `io.micronaut.flyway:micronaut-flyway:6.0.0-M2` uses flyway `9.20.1` and not `9.16.1`.

We can solve this by forcing the flyway version
```suggestion
    implementation("io.micronaut.flyway:micronaut-flyway")
    implementation("org.flywaydb:flyway-core") {
        version {
            strictly("9.16.1")
        }
    }
```